srv/components/provisioner/salt_master/files/master

```diff
@@ -850,8 +851,8 @@ gitfs_remotes:
 #  - cmd_yaml: cat /etc/salt/yaml
 pillar_roots:
   base:
-    - /var/lib/seagate/cortx/provisioner/local/srv/pillar  # new style
```

Comment:
why do you need to remove that? it was set to support future enhancement when local pillar might be set to be more prioritized over shared ones.

Reply:
I've added it back